Dockerfile.ubuntu_builder

```diff
@@ -0,0 +1,46 @@
+# syntax=docker/dockerfile:1
+
+ARG BASE=12.2.0-devel-ubuntu22.04
+FROM nvidia/cuda:${BASE} AS build
+
+LABEL maintainer="B.D.Evans@sussex.ac.uk" \
+    org.opencontainers.image.documentation="https://genn-team.github.io/" \
+    org.opencontainers.image.source="https://github.com/genn-team/genn" \
+    org.opencontainers.image.title="PyGeNN wheel builder"
+
+ARG PY_VER=3.11
+# Update APT database and upgrade any outdated packages and install Python and pip
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -yq --no-install-recommends python${PY_VER}-dev python3-pip
+
+# Set environment variables
+ENV CUDA_PATH=/usr/local/cuda \
+    GENN_PATH=/opt/genn
+
+# Set python3 to be the default version of python
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python${PY_VER} 1
+
+# Upgrade pip itself and install numpy and swig
+RUN python -m pip install --upgrade pip && \
+    python -m pip install numpy swig
+
+# Copy GeNN into /opt
+COPY . ${GENN_PATH}
+
+# Use this as working directory
+WORKDIR ${GENN_PATH}
+
+# Install GeNN and PyGeNN
+RUN make install -j `lscpu -p | egrep -v '^#' | sort -u -t, -k 2,4 | wc -l`
```

Comment:
there is no need to ``make install`` when all you're doing is build PyGeNN - that's only if you want a system-wide installation to use GeNN from C++

Reply:
Ok I'll remove it.